src/components/CtaSection/Cta.jsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import imgAmy from '../../assets/image-amy.webp';
+import './Cta.scss';
+
+const Cta = () => {
+    return (
+        <div className="cta">
+            <img className="cta_image" src={imgAmy} alt="" />
```

Comment:
I think you used the double underscore on some previous commit. which is great! it's the BEM convention. We should definetly keep using it :)

Reply:
Okay. Sure. Makes sense